modules/party/price_edit.php

```diff
@@ -22,5 +22,6 @@
 $mf->AddField(t('Sichtbar für'), 'requirement', \LanSuite\MasterForm::IS_SELECTION, $selectrequire, 1);
 $mf->AddField(t('Gültig bis'), 'enddate');
 
-$mf->SendForm('index.php?mod=party&action=price_edit&party_id='. $_GET['party_id'], 'party_prices', 'price_id', $_GET['price_id']);
+$priceIDParameter = $_GET['price_id'] ?? 0;
```

Comment:
If the party id is 0, it will get the party id from the global var `$party`, see
https://github.com/lansuite/lansuite/blob/25e08dbb1f86f02daa7f30537386621a4944b70a/modules/party/price_edit.php#L8-L10

And this `party_id` comes from a chain of possibilities, see https://github.com/lansuite/lansuite/blob/25e08dbb1f86f02daa7f30537386621a4944b70a/modules/party/Classes/Party.php#L28-L49

There is a high chance, that this is ok and a party has a reference.